docs/source/commands/dev.mdx

```diff
@@ -1,88 +1,108 @@
 
-> ⚠️Rover uses the port of the running router to identify an existing session, so if you are defining a port via `--supergraph-port` or `--router-config`, make sure to use the same port for all `rover dev` processes that you want to attach to the same session.
+If you have a standard set of subgraphs that you're always developing with, you can create a [supergraph config file](./supergraphs#yaml-configuration-file) to add _all_ of them to your initial session with a single command.
 
-## Connecting multiple subgraphs at once
+For example, this `supergraph.yaml` file provides the necessary details for two subgraphs:
 
-If you have a standard set of subgraphs you are always developing with, you can use a [supergraph config file](/rover/commands/supergraphs#yaml-configuration-file) to connect them all at once. For example, if you have a `supergraph.yaml` file with the following contents:
-
-```yaml
+```yaml title="supergraph.yaml"
 federation_version: =2.4.7
 subgraphs:
   products:
     routing_url: http://localhost:4000
     schema:
-      file: ./products.graphql
+      file: ./products.graphql # Schema provided via file
   reviews:
     routing_url: http://localhost:4001
     schema:
-      subgraph_url: http://localhost:4001
+      subgraph_url: http://localhost:4001 # Schema provided via introspection
+```
+
+You provide this file to `rover dev` like so:
+
+```bash
+rover dev --supergraph-config supergraph.yaml
 ```
 
-You can run a single `rover dev --supergraph-config supergraph.yaml` command to start a session with both subgraphs (equivalent to running the two separate commands from the above sections). This also enables you to use other features of the `supergraph.yaml` file like `introspection_headers`. You can still add more subgraphs to the session using `rover dev` in a separate terminal, but only the _first_ session can use a `supergraph.yaml` file.
+If you do, a `rover dev` session starts with _one_ of the subgraphs listed, then adds the remaining subgraphs one at a time (order is undefined). Because of this, you might observe composition errors during intermediate steps.
+
+> Providing a `supergraph.yaml` file also enables you to take advantage of [other config options](./supergraphs#yaml-configuration-file), such as `introspection_headers`.
+
+After you start your initial `rover dev` session with a config file, you can still [add other subgraphs individually](#adding-a-subgraph-to-a-session). However, you _can't_ provide another config file.
+
+## Adding a subgraph to a session
+
+After you start a `rover dev` session with one subgraph, you can then add _other_ subgraphs to that same session.
+
+To add a subgraph, open a new terminal window and run `rover dev` _again_, this time providing the details of the subgraph to add. For example, this command adds a `reviews` subgraph to the session started with a `products` subgraph above:
+
+```bash
+rover dev --name reviews --url http://localhost:4001
```

Comment:
Since `reviews` in this scenario would conflict with `reviews` added via config
```suggestion
To add a subgraph, open a new terminal window and run `rover dev` _again_, this time providing the details of the subgraph to add. For example, this command adds a `users` subgraph:

```bash
rover dev --name users --url http://localhost:4002
```